internal/pkg/cli/pipeline_init.go

```diff
@@ -225,6 +242,51 @@ func (o *initPipelineOpts) validateURL(url string) error {
 	return nil
 }
 
+func (o *initPipelineOpts) validateURLExists(url string) error {
```

Comment:
Maybe we should mention it is for local git. Same for `o.fetchAndParseURLs`

Reply:
I'll add to the docs! Thanks.